Makefile.pre.in

```diff
@@ -691,12 +692,14 @@ regen-importlib: Programs/_freeze_importlib
 	# from Lib/importlib/_bootstrap_external.py using _freeze_importlib
 	./Programs/_freeze_importlib \
 	    $(srcdir)/Lib/importlib/_bootstrap_external.py \
-	    $(srcdir)/Python/importlib_external.h
+	    $(srcdir)/Python/importlib_external.h.new
+	$(UPDATE_FILE) $(srcdir)/Python/importlib_external.h $(srcdir)/Python/importlib_external.h.new
 	# Regenerate Python/importlib.h from Lib/importlib/_bootstrap.py
 	# using _freeze_importlib
 	./Programs/_freeze_importlib \
 	    $(srcdir)/Lib/importlib/_bootstrap.py \
-	    $(srcdir)/Python/importlib.h
+	    $(srcdir)/Python/importlib.h.new
+	$(UPDATE_FILE) $(srcdir)/Python/importlib.h $(srcdir)/Python/importlib.h.new
```

Comment:
It's a little bit boring to always have to repeat the full path. Why not passing a single parameter to the update script, and add ".new" to get the "new filename"?

Reply:
Because it doesn't impose a naming convention and makes it explciit which files are operated on. Also, there's a lot of boredom in Makefiles ;-)

Reply:
Ok. It was just a suggestion, being explicit is fine.

> Also, there's a lot of boredom in Makefiles ;-)

I will not start discussing that aspect of Makefiles, thanks.